### DISCUSSION TITLE
[Feature] Assign face to existing person

### DISCUSSION THREAD

**Discussion by fitim7** [immich-app/immich#13168] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Like the title is saying: at the moment I am missing the feature of assigning a face to an existing person in the app. You can assign faces just to new names, which is often not correct. In the web Immich is proposing already existing names when they are similar. This would be great for the app too.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6259
